### ISSUE TITLE
Add Linting as a part of CI pipeline

### ISSUE THREAD

**Issue by tarunsinghofficial** [GoogleChromeLabs/web-audio-samples#392]
## Overview
We should add automated linting to our Continuous Integration (CI) pipeline to ensure consistent code style and catch potential errors early in the development process.

## Current Situation
- Currently, there is no automated linting in our CI pipeline.
- Code style inconsistencies are being caught manually during code reviews.
- We have `.eslintrc.json` and `.prettierrc.json` files in our repository, but they are not enforced automatically.

## Proposed Solution
Add a linting step to our GitHub Actions workflow that runs on every pull request and push to main branches.

## Implementation Details
1. Create or update a GitHub Actions workflow file (e.g., `.github/workflows/ci.yml`).
2. Add a linting job that includes:
   - Setting up Node.js
   - Installing dependencies
   - Running ESLint
   - (Optional) Running Prettier

**Comment by tarunsinghofficial** [comment]
I'm interested in doing this issue. Let me know if it is good, and please assign the issue to me. Thanks!

**Comment by hoch** [comment]
Sure thing. I'll assign this to you.

**Comment by tarunsinghofficial** [comment]
> Sure thing. I'll assign this to you.

Hi @hoch 

Please review PR #393, whenever you're free. Thanks :)

**Comment by hoch** [comment]
Yes. We will get to this soon after wrapping up the https://github.com/GoogleChromeLabs/web-audio-samples/pull/395. Thanks for your contribution!
